xmtp_db/src/encrypted_store/database/native/sqlcipher_connection.rs

```diff
@@ -66,7 +66,7 @@ impl EncryptedConnection {
                             db_pathbuf.display(),
                             salt_path.display(),
                         );
-                        let file = File::open(salt_path)?;
+                        let file = BufReader::new(File::open(salt_path)?);
```

Comment:
is this addressing any specific issue? seems unrelated to rest of the PR.

Reply:
Clippy complained that opening without a buffer is slow